en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+* Add the new issue in the "Backlog" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1).
+* Use the [CVSS Calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) to determine the issue severity.
+* Once the issue is confirmed, create a _draft security advisory_ against the relevant GitHub repository. This will allow you to request a CVE.
+* Once a CVE has been assigned, respond to issue reporter and add it to the Github issue.
+* Ask the reporter if they want to be credited for the disclosure and under what name.
+* Clarify who picks up and owns the issue (assign in Github).
+  The owner can be separate from the developer resolving the issue,
+  their primary responsibility is to ensure the issue keeps moving through the process correctly.
+
+### When developing a fix
+
+* Ensure you're working on the oldest supported minor release branch of every supported major release (see [Supported Versions](#supported-versions))
+* Move the issue into "In Progress" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+* Add fixes on the [http://github.com/silverstripe-security](http://github.com/silverstripe-security) repo. Don't forget to update branches from the upstream repo.
+* Ensure that all security commit messages are prefixed with the CVE. E.g. "[CVE-2019-001] Fixed invalid XSS"
+
+### Before release
+
+* For issues rated "high" or "critical" (CVSS of >=7.0), post a pre-announcement to the [security pre-announcement list](https://groups.google.com/a/silverstripe.com/forum/#!forum/security-preannounce).
+  It should include a basic "preannouncement description" which doesn't give away too much,
+  the CVSS score as well as the CVE identifier.
+* Create a draft page under [Open Source > Download > Security Releases](https://www.silverstripe.org/admin/pages/edit/show/794).
+  Populate it with the information from the [Github project board](https://github.com/silverstripe-security/security-issues/projects/1).
+* Link to silverstripe.org security release page in the changelog.
+* Update the _draft security advisory_ on GitHub.
+* Move the issue to "Awaiting Release" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+ 
+### After release
+
+* Publish silverstripe.org security release page
+* Respond to issue reporter with reference to the release on the same discussion thread (cc security@silverstripe.org)
+* Publish the security advisory on GitHub. This will also publish the CVE.
+* Open a pull request to [https://github.com/FriendsOfPHP/security-advisories/tree/master/silverstripe](https://github.com/FriendsOfPHP/security-advisories/tree/master/silverstripe) repository.
+* Move the issue to "Done" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+
+### Detailed CVE and CVSS Guidance
```

Comment:
Probably this should be linked to (or at least mentioned) in the steps above that would benefit from this extra context.

Reply:
Where did this go? I can't find it in the current page?

Reply:
I've renamed it "Creating a GitHub security advisory". This section used to be about how to request a CVE from Mitre ... the previous title seemed like it wasn't really descriptive with the new context. Happy to revert it if you disagree.

Reply:
The new name is fine, I just didn't know you'd changed it is all.